docs/reference/contrib/modeladmin/primer.rst

```diff
@@ -142,15 +142,20 @@ Overriding templates
 --------------------
 
 For all modeladmin views, Wagtail looks for templates in the following folders
-within your project, before resorting to the defaults:
+within your project before resorting to the defaults:
 
 1. ``/modeladmin/app-name/model-name/``
 2. ``/modeladmin/app-name/``
 3. ``/modeladmin/``
```

Comment:
The updates here are helpful but also potentially add some confusion.

the following locations will all work (using the bakerydemo project)
* `bakerydemo/templates/modeladmin/index.html`
* `bakerydemo/breads/templates/modeladmin/breads/index.html`

In the case above, the default modeladmin index will now be the one in the root templates folder, but any modeladmin within breads app will use the other one.

This change kind of implies you must always put the templates folder within a Django app, when you should also be able to use the root templates folder.

Maybe... this first list needs to be updated to reflect that templates should be in a `templates` folder, but not sure if that will make it even more confusing.

It would be good to try to explain the hierarchy a bit better with these docs changes.

Reply:
> Maybe... this first list needs to be updated to reflect that templates should be in a templates folder, but not sure if that will make it even more confusing

I think this would make a lot of sense!